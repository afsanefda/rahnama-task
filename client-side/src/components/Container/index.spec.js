import React from 'react';
import renderer from 'react-test-renderer';
import { Container } from '..';

describe('<Container /> test', () => {
  let component;

  beforeEach(() => {
    component = (
      <Container />
    );
  });

  it('container should renders correctly - snapshot', () => {
    const tree = renderer.create(component).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
