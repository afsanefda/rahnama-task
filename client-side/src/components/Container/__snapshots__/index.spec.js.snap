// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Container /> test container should renders correctly - snapshot 1`] = `
<div
  className="container"
/>
`;
